api/test/wfl/tools/fixtures.clj

```diff
@@ -42,79 +37,49 @@
               (gcs/list-objects gcs-test-bucket name#)
               (run! (comp delete-test-object :name)))))))
 
-(defn cloud-db-url
-  "Return NIL or an URL for the CloudSQL instance containing DB-NAME."
-  [environment instance-name db-name]
-  (if-let [gcp (get-in env/stuff [environment :server :project])]
-    (letfn [(postgresql? [{:keys [instanceType name region]}]
-              (when (= [instanceType name]
-                      ["CLOUD_SQL_INSTANCE" instance-name])
-                (str "jdbc:postgresql://google/" db-name "?useSSL=false"
-                  "&socketFactory="
-                  "com.google.cloud.sql.postgres.SocketFactory"
-                  "&cloudSqlInstance="
-                  (str/join ":" [gcp region name]))))]
-      (-> {:method  :get
-           :url     (str "https://www.googleapis.com/sql/v1beta4/projects/"
-                      gcp "/instances")
-           :headers (merge {"Content-Type" "application/json"}
-                      (once/get-auth-header))}
-        http/request
-        :body
-        (json/read-str :key-fn keyword)
-        :items
-        (->> (keep postgresql?))
-        first))))
-
-(def testing-dbname
-  (:db-name (postgres/wfl-db-config)))
-
-(defn- postgres-db-config []
-  (->
-    (postgres/wfl-db-config)
+(defn ^:private postgres-db-config []
+  (-> (postgres/wfl-db-config)
     (dissoc :instance-name)
     (merge {:connection-uri "jdbc:postgresql:postgres"
             :db-name        "postgres"})))
 
+(def testing-db-config
+  (let [name (str "wfltest" (str/replace (UUID/randomUUID) "-" ""))]
+    (-> (postgres/wfl-db-config)
+      (dissoc :instance-name)
+      (merge {:connection-uri (str "jdbc:postgresql:" name)
+              :db-name        name})
+      constantly)))
```

Comment:
Why `constantly` rather than just a `defn`?
(Does JDBC use :instance-name in the default case?)

Reply:
I'm not sure what jdbc uses. I only removed this because it looks specifically cloud sql related (rather, specific to our cloud sql database).

Reply:
I'm using `constantly` so that this name has the same "feel" as `wfl-db-config` (see workloads.clj in this change). 
Since I'm appending a random uuid to the db name, I need this data to be constant throughout the test.